README.md

```diff
@@ -1,3 +1,49 @@
 # ecomm
 
-Projeto de Ecommerce criando durante o programa LevelUp da Alura
\ No newline at end of file
+Projeto de Ecommerce criando durante o programa LevelUp da Alura
```

Comment:
Tente utilizar as tags do Markdown para formatar sua documentação: https://docs.github.com/pt/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax